### DISCUSSION TITLE
Manually sync buttons

### DISCUSSION THREAD

**Discussion by spc789** [damionx7/Zettel-Notes-Documentation#104] [Feature Requests]
Add buttons to manually sync a repo (ie manually upload the repo or manually download a repo from a backup source). For example : one edited a note in zettel notes and needs it urgently on some other place (for example on a desktop) an instant push (sync) can be needed...

**Comment by aamott** [comment]
You can simply pull down on the repo to sync notes. *However* you may run into the issue from #29. It would be nice if it were a button too though, maybe in the kebab menu? It took me some time to realize the obvious "pull to reload" was it.

**Comment by damionx7** [comment]
You can use widgets to trigger synchronization in pull or push mode.

![Screenshot_2024-06-26-20-59-08-43_377fe8767025e4fa6dddb62d6df8f716.jpg](https://github.com/damionx7/Zettel-Notes-Documentation/assets/6530843/e873180a-930d-48d5-9143-125db5ff8ee2)
